src/layout/Header.tsx

```diff
@@ -1 +1,61 @@
-export const Header = () => <div className="font-bold">Headerrr</div>;
+import { SocialIcon } from './components/SocialIcon';
+import { NavHeaderLink } from './components/NavLinks';
+import { useState } from 'react';
+import { faViber, faFacebook, faInstagram } from '@fortawesome/free-brands-svg-icons';
+import Logo from '../images/VilaMarina.png';
+
+export const Header = () => {
+   const [menuOpen, setMenuOpen] = useState(false);
+
+   const toggleMenu = () => {
+      setMenuOpen(!menuOpen);
+   };
+
+   return (
+      <div className="py-3 bg-sky-500 shadow-md">
+         <div className="nav container mx-auto px-2">
+            <div className="content flex justify-between">
+               <div className="logo cursor-pointer hover:-translate-y-2 hover:drop-shadow-md duration-300 z-20">
+                  <img src={Logo} className="lg:w-20 lg:h-20 w-14 h-14" alt="Vila Marina Jahorina Logo" />
+               </div>
+               <div className="navbar flex gap-8 items-center">
+                  <div className="icon-menu hidden md:block">
+                     <ul className="icon-menu-content items-center flex gap-4">
+                        <SocialIcon href="facebook link" icon={faViber} />
+                        <SocialIcon href="facebook link" text="B." extraClasses="text-[38px]  font-bold relative top-[-6px]" />
```

Comment:
Does this even work? I believe the pattern is like this:
```suggestion
                        <SocialIcon href="facebook link" text="B." extraClasses="text-[38px]  font-bold relative -top[-6px]" />
```